packages/core/tsup.config.ts

```diff
@@ -5,7 +5,7 @@ export const tsup: Options = {
   sourcemap: true,
   clean: true,
   entryPoints: ['src/index.ts'],
-  format: ['esm', 'cjs'],
+  format: ['esm'],
```

Comment:
We'll see how this fairs for using CJS-forward environments like Next.js or server-side/lambda functions. 

This MR should now allow the use of `.mjs` config files which should cause Node v16+ environments to treat Flatbread as a module.